api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -424,116 +424,117 @@ uiConfiguration: >-
     },
     "criteriaConfigs": [{
       "type":"concept",
+      "id": "tanagra-conditions",
```

Comment:
Just curious, why the "tanagra-" prefix?

Reply:
Plugin criteria ids would live in the same namespace so I wanted to avoid collisions on such common names.